src/views/SystemList/Table.tsx

```diff
@@ -219,27 +219,23 @@ export const Table = ({
           : -1
     });
 
-    /*
-    {
-      Header: t<string>('systemTable.header.systemStatus'),
-      accessor: 'status',
-      id: 'systemStatus',
-      disableGlobalFilter: true,
-      Cell: ({ row }: { row: Row<CedarSystem> }) => (
-        <div>
-          <SystemHealthIcon
-            status={mapCedarStatusToIcon(row.original.status)}
-            size="medium"
-            className="margin-right-1"
-          />
-          <span>{row.original.status}</span>
-        </div>
-      )
+    if (isHomePage) {
+      cols.push({
+        Header: t<string>('systemTable.header.openRequests'),
+        accessor: (system: CedarSystem) => {
+          return (
+            system.linkedTrbRequests.length + system.linkedSystemIntakes.length
+          );
+        },
+        id: 'openRequests',
+        Cell: ({ row }: { row: Row<CedarSystem> }) =>
```

Comment:
You can remove the Cell assignment entirely now that accessor is defined.
This is ready to approve after that.

Reply:
nice! thanks! fixed